rules/no-uat-phase-in-scrum/rule.md

```diff
@@ -0,0 +1,59 @@
+---
+seoDescription: Embrace UAT in Scrum by incorporating your testing processes into the Sprint 
+type: rule
+title: Do you include UAT in each Sprint?
+uri: no-uat-phase-in-scrum
+authors:
+  - title: Luke Cook
+    url: https://www.ssw.com.au/people/luke-cook
+  - title: Gordon Beeming
+    url: https://www.ssw.com.au/people/gordon-beeming
+related:
+  - acceptance-criteria
+  - do-you-know-which-environments-you-need-to-provision-when-starting-a-new-project
+  - do-you-have-separate-development-testing-and-production-environments
+created: 2025-10-02T22:40:45.273Z
+guid: 091b9044-5067-4bfc-aa15-0fcd150eb94b
+---
+
+User Acceptance Testing (UAT) is an integral component of software release cycles, but it can often be treated as a secondary backlog of work that's managed independently of the development Sprint. This separation adds friction, creates delays, and **costs you more money** for every piece of work. UAT is incredibly valuable, so make sure you're doing it right.
+
+<!-- endintro -->
+
+![Figure: Break down the wall between development and testing](dev-uat.png)
+
+## What is UAT?
+
+Developers build a feature, and before shipping that feature to the wider audience, you want to make sure it's doing exactly what you expect. This is the foundation of UAT. Depending on the size and scope of the project, UAT may be done by a single Product Owner, or a team of dedicated testers. Regardless; the concept remains the same. Once the development work has been signed off by the relevant testing authority, the work is approved for production and is ready to be shipped.
+
+## UAT done wrong
+
+In many projects, QA or testing will be treated as an entirely separate phase to development. The Scrum team will work on a Product Backlog Item, merge their code, and set the PBI's status to "Ready for testing". These PBIs will then be managed separately as a "UAT backlog", where it may be days or weeks until that PBI is picked up by the QA team and actioned.
+
+Additionally, there may be bugs found by testers, meaning those PBIs are thrown back onto the development backlog and have to be re-prioritized and scheduled for future Sprints (or, worse, disrupt the current Sprint). Not to mention that, after all this time, the developers themselves may have already context-switched to different problems and will have to reacquaint themselves with the feature once again.
+
+As you can imagine, this disconnect between development and acceptance testing can result in a significant drop in delivery speed, and dramatically increase the business cost for many work items.
```

Comment:
Can you add a comment like "A PBI marked as "Ready for Testing" effectively means it's not truly "Done" by the end of the Sprint. For the Sprint Goal to be met, every PBI must be able to satisfy the team's complete Definition of Done, irrespective of the specific workflow steps defined."

Reply:
Done. Check you're happy with the wording though pls